### DISCUSSION TITLE
The path could not be found. Please ensure the path is valid and try again.

### DISCUSSION THREAD

**Discussion by paivadaniel** [jellyfin/jellyfin#9658] [Q&A]
I installed Jellyfin and use Ubuntu Desktop 22.04, however, when adding a library I can't set the subfolders of /media (/media/Photos, where Photos is a mounted hdd) appear (https://i.imgur.com/IHzvrjq.png), where my external HDs are mounted with the movies. I also entered my server address in that setting, but it didn't help. The /media folder is configured with chmod 755 and username/group name for my user, not root, strangely Jellifyn lists some subdirectories of /etc, which requires root user, and does not list the subdirectories of /media.

When I try to type /media/Photos, the program returns an error: 

"The path could not be found. Please ensure the path is valid and try again."

I found this topic about the same problem: https://www.reddit.com/r/jellyfin/comments/gaojft/the_path_could_not_be_found_please_ensure_the/

But the solution (sudo setfacl -m u:jellyfin:rx /media/Fotos/ or 
sudo setfacl -m u:myuser:rx /media/Fotos/) did not work to me.

**Comment by npe3uDeHT** [comment]
Have same problem. Jellyfin - 10.8.10, OS - Gentoo, kernel - 6.1.3, directory - /home (chmod 755, chown root:root), directory /home/Photo (chmod 755, chown jellyfin:jellyfin).
https://i.imgur.com/2gZA3u6.jpeg
https://i.imgur.com/BFqUSs6.jpeg

Solution (setfacl -m u:jellyfin:rx /home/Photo) did not work for me too.

**Comment by Nexxsys** [comment]
Same for me.  Unfortunate you can't go more than 2 levels in sub directories.  The Solution didn't work for me either.

**Comment by ohero636** [comment]
This solution is working for me, 
if you want to share /media/USER/Drive
be sure to add the subfolder where your drive is ex: /media/USER 



MYGROUP="$USER"
sudo usermod -a -G $MYGROUP jellyfin
sudo setfacl -m  u:jellyfin:rx /media/USER/     
sudo setfacl -m  g:jellyfin:rx /media/USER/

**Reply by thamulot** [reply]
It worked for me wonder. I would suggest this to be the answer. Other than that, Jellyfin should do a better job in the doc at least since nowhere it is explained there is a `jellyfin` user created (at least, when installing on Debian with the default service config) and that you must giving it rights on your media folder.

This whole subject deserves its entire section since it gets complicated fast - for example vs docker mount, vs filesystem, etc.

**Reply by madmadworldmediacorp** [reply]
I should have come back and updated the thread. I got everything working. It's just a quirk with Plex on Linux.

**Reply by bnobu** [reply]
This worked for me, too.

**Comment by kshockk64** [comment]
I am using Ubuntu server 24.04 and Jellyfin and had same problem. I had to go in to fstab and change mount point to:
/dev/sdd /mnt/Movies ext4 defaults 0 0
Check in disk manager to see what /dev/sa? you are trying to mount and enter the path to you folder, your /media/Photos would become /mnt/Photos. This is assuming your disk is formatted as ext4.
